windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -173,9 +173,17 @@ $replacement = (Get-Content $replacement_file) -join("`r`n")
 
 
 ##############################
-#  STEP 5 - REMOVE UNNECESSARY FILES.
+#  STEP 4.2 - COPY NECESSARY DLL FILES.
 ##############################
+# Save the C++ runtime DLL to allow running gems with C++ native extensions
+# included gem such as winevt_c.
```

Comment:
"included gem" is now superfluous.

Reply:
The dll is for winevt_c's use. winevt_c is introduced by windows_eventlog2. And it was the reason of LoadError. I want to leave the linking message here so that pointing this dll to winevt_c.

Reply:
Do you mean i need to remove `included gem`? 

but if i append `such as winevt_c` to the end of `native extensions`. The words distance between the `winevt_c` to the `running gems` is too far. ppl might think that `winevt_c` is a native extensions instead of a gem.........

Reply:
To be clear, I was simply suggesting the following phrasing:
```
# Save the C++ runtime DLL to allow running gems with C++ native extensions
# such as winevt_c.
```
(because the first line already mentions "gems").

Reply:
See my above comments and paste it again below

```
Do you mean i need to remove included gem?

but if i append such as winevt_c to the end of native extensions. The words distance between the winevt_c to the running gems is too far. ppl might think that winevt_c is a native extensions instead of a gem.........
```

Reply:
Done